telethon/client/messages.py

```diff
@@ -1342,6 +1343,11 @@ async def pin_message(
 
             notify (`bool`, optional):
                 Whether the pin should notify people or not.
+                
+            pm_oneside (`bool`, optional):
+                Whether the message should be deleted for everyone or not.
```

Comment:
Just a Typo
Would be better,  If you make "deleted" to "pinned"

Reply:
sorry. i copy pasted from deleted_messages docs.